cmd/utils/flags.go

```diff
@@ -897,6 +902,26 @@ func MakeDataDir(ctx *cli.Context) string {
 	return ""
 }
 
+// MakeRaftLogDir retrieves the currently requested data directory, terminating
+// if none (or the empty string) is specified. If the node is starting a testnet,
+// the a subdirectory of the specified raflogdir will be used.
+func MakeRaftLogDir(ctx *cli.Context) string {
+	if path := ctx.GlobalString(RaftLogDirFlag.Name); path != "" {
+		if ctx.GlobalBool(TestnetFlag.Name) {
+			return filepath.Join(path, "testnet")
+		}
+		if ctx.GlobalBool(RinkebyFlag.Name) {
+			return filepath.Join(path, "rinkeby")
+		}
+		if ctx.GlobalBool(GoerliFlag.Name) {
+			return filepath.Join(path, "goerli")
+		}
+		return path
+	}
+	Fatalf("Cannot determine default raft log directory, please set manually(--raftlogdir)")
+	return ""
+}
```

Comment:
This function doesn't look like it's used anywhere. If so, it should be removed.

Reply:
Yep, this function is not neccessary.